tests/mocks/mock-ordinalscom-api.ts

```diff
@@ -0,0 +1,46 @@
+export const mockOrdinalscomApiHtmlResponse = `
```

Comment:
Just noticed in other PR you typ used `...OrdinalsCom...` so noting here you might want to capitalize `Com` to make code searches consistent?